### PR TITLE
Release 3.6.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.6.0 (2019-05-03)
+
+- Added mixins for responsive desings. [#58](https://github.com/blackbaud/skyux-theme/pull/58)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Small typo. designs

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 3.6.0 (2019-05-03)
+
+- Added mixins for responsive designs. [#58](https://github.com/blackbaud/skyux-theme/pull/58)
+- Fixed radio and checkbox labels to have correct spacing. [#62](https://github.com/blackbaud/skyux-theme/pull/62)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Sorry, I don't know why I didn't include this in the first comment. 😬 

Fixed spacing for radio and checkbox labels.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: No problem. Done.

**Pull request merged**